sql/_23_apricot_qa/_03_i18n/tr_TR/_09_identifiers/answers/_007_identifiers_index_cardinality.answer_cci

```diff
@@ -18,21 +18,21 @@
-��_��_��_��_��_��     1     i_��_��_��_��_��_��_��     1     ��     D     0     null     null     YES     BTREE     null     null     YES     
-��_��_��_��_��_��     1     i_��_��_��_��_��_��_��_��     1     ��     A     0     null     null     YES     BTREE     null     null     YES     
-��_��_��_��_��_��     1     i_��_��_��_��_��_��_��_��     2     ��     A     0     null     null     YES     BTREE     null     null     YES     
-��_��_��_��_��_��     1     i_��_��_��_��_��_��_��     1     ��     A     0     7     null     YES     BTREE     null     null     YES     
-��_��_��_��_��_��     0     i_��_��_��_��_��_��_��_��     1     ��     A     0     null     null     NO     BTREE     null     null     YES     
-��_��_��_��_��_��     0     i_��_��_��_��_��_��_��_��     2     ��     A     0     null     null     YES     BTREE     null     null     YES     
-��_��_��_��_��_��     0     u_��_��_��_��_��_��_��     1     ��     A     0     null     null     YES     BTREE     null     null     YES     
-��_��_��_��_��_��     0     u_��_��_��_��_��_��_��     1     ��     A     0     null     null     YES     BTREE     null     null     YES     
+dba.��_��_��_��_��_��     1     i_��_��_��_��_��_��_��     1     ��     D     0     null     null     YES     BTREE     null     null     YES     
+dba.��_��_��_��_��_��     1     i_��_��_��_��_��_��_��_��     1     ��     A     0     null     null     YES     BTREE     null     null     YES     
+dba.��_��_��_��_��_��     1     i_��_��_��_��_��_��_��_��     2     ��     A     0     null     null     YES     BTREE     null     null     YES     
+dba.��_��_��_��_��_��     1     i_��_��_��_��_��_��_��     1     ��     A     0     7     null     YES     BTREE     null     null     YES     
+dba.��_��_��_��_��_��     0     i_��_��_��_��_��_��_��_��     1     ��     A     0     null     null     NO     BTREE     null     null     YES     
+dba.��_��_��_��_��_��     0     i_��_��_��_��_��_��_��_��     2     ��     A     0     null     null     YES     BTREE     null     null     YES     
+dba.��_��_��_��_��_��     0     u_��_��_��_��_��_��_��     1     ��     A     0     null     null     YES     BTREE     null     null     YES     
+dba.��_��_��_��_��_��     0     u_��_��_��_��_��_��_��     1     ��     A     0     null     null     YES     BTREE     null     null     YES     
 
 ===================================================
-index_cardinality(_utf8'Çç_Ğğ_İı_Öö_Şş_Üü', _utf8'i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö', 0)    
+index_cardinality(_utf8'dba.Çç_Ğğ_İı_Öö_Şş_Üü', _utf8'i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö', 0)    
 0     
 
 ===================================================
-index_cardinality(_utf8'Çç_Ğğ_İı_Öö_Şş_Üü', _utf8'i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö', 1)    
+index_cardinality(_utf8'dba.Çç_Ğğ_İı_Öö_Şş_Üü', _utf8'i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö', 1)    
 0     
 
 ===================================================
```

Comment:
As the below file is changed...
```
sql/_23_apricot_qa/_03_i18n/tr_TR/_09_identifiers/cases/_007_identifiers_index_cardinality.sql
```
The column name of the result below should also be changed.
```
===================================================
index_cardinality(_utf8'dba.Çç_Ğğ_İı_Öö_Şş_Üü', _utf8'i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö', 2)    
null     

===================================================
index_cardinality(_utf8'dba.Çç_Ğğ_İı_Öö_Şş_Üü23', _utf8'i_Çç_Ğğ_İı_Öö_Şş_Üü_Çç_Öö', 1)    
null     
```

Reply:
Reflected the review. Please check again.